scripts/sanitycheck

```diff
@@ -2023,6 +2023,9 @@ def parse_arguments():
         "the new app size is greater then the specified percentage "
         "from the last release. Default is 5. 0 to warn on any "
         "increase on app size")
+    parser.add_argument(
```

Comment:
Is this part of the patch?  (The description only mentions `__init__.py`)

Reply:
duh